### PR TITLE
[ADZ-1184] 

### PR DESCRIPTION
**Pull request by harryhmc**
Contains changes for the document list and side-nav sections (note: filter section not part of this PR) for ADZ-1184 and ADZ-1182.

### CODE REVIEW

File: repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl [hunk 1]
```diff
@@ -18,46 +17,33 @@
...
-        </div>
+<@headerBanner document />
 
-        <#assign alphabetical_hash = group_blocks(flat_blocks(apiCatalogueLinks true))/>
+<div class="nhsd-t-grid nhsd-!t-margin-top-6">
+
+    <div class="nhsd-t-row">
+        <div class="rich-text-content nhsd-t-col-12">
+            <#if document.body?has_content??>
+                <@hst.html hippohtml=document.body contentRewriter=gaContentRewriter/>
```
Comment (LEJA3) [inline repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl:29]: The `brContentRewriter` content rewriter should be used here to correctly style CMS content.

File: repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl [hunk 2]
```diff
@@ -18,46 +17,33 @@
...
-                <#-- [FTL-END] 'Summary and optional Body' sections -->
-            </div>
-        </div>
+<@headerBanner document />
 
-        <#assign alphabetical_hash = group_blocks(flat_blocks(apiCatalogueLinks true))/>
+<div class="nhsd-t-grid nhsd-!t-margin-top-6">
+
+    <div class="nhsd-t-row">
+        <div class="rich-text-content nhsd-t-col-12">
```
Comment (LEJA3) [inline repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl:27]: Class `rich-text-content` isn't needed.

**Review by LEJA3 — COMMENTED** [review]
I can see the filter nav is still to be redesigned so I haven't commented on that.

File: repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl (hunk 1 above)
Comment (harryhmc) [reply repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl]: Thanks Lee, have replaced it.

File: repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl (hunk 2 above)
Comment (harryhmc) [reply repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl]: Removed :+1:

File: repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl [hunk 3]
```diff
@@ -18,46 +17,33 @@
...
 
-                <div class="column column--two-thirds page-block page-block--main">
-                    <@apiCatalogueEntries alphabetical_hash filtersModel></@apiCatalogueEntries>
-                    <div class="article-section muted">
-                        <@lastModified document.lastModified false></@lastModified>
-                    </div>
-                </div>
+    <#if alphabetical_hash??>
+    <div class="nhsd-t-row">
+        <div class="nhsd-t-col-3 nhsd-!t-display-hide nhsd-!t-display-l-show sticky sticky--top">
```
Comment (LEJA3) [inline repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl:37]: Classes `sticky` & `sticky--top` are not needed

**Review by LEJA3 — APPROVED** [review]

**Pull request merged**